packages/react-scripts/scripts/new-component/templates/actions.test.tmpl

```diff
@@ -1,6 +1,6 @@
 import * as actions from './actions';
 
-describe('Teast actions', () => {
+describe('{{ component }} actions', () => {
```

Comment:
I saw this just this morning one on of my components. I thought I was something I did.